benchmarks/tpc-ds/ds_refresh.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Sequentially run the first refresh run for tpc-ds and then verify the results.",
+  "query_files": [
+    "queries/refresh/LF_CR.sql",
+    "queries/verify_LF_CR.sql"
+  ]
+}
```

Comment:
any expected number of rows returned?

Reply:
so the expected number of rows is actually an expression like >=2. Do we support such expressions currently @wanglinsong ?

Reply:
How do you tell if the verification is successful?

Reply:
So there is a way to compare the output with a baseline result and is currently being used for ds_orderby flavor. We will extend that support for ds_refresh flavor as well.